merge-all.rb

```diff
@@ -57,7 +57,7 @@ def branch_name
   if cocina_level2
     COCINA_LEVEL2_BRANCH_NAME
   else
-    BRANCH_NAME
+    ENV['BRANCH_NAME'] || BRANCH_NAME
```

Comment:
What would you think about moving the logic to line 11? Like:

```ruby
BRANCH_NAME = ENV.fetch('BRANCH_NAME', 'update-dependencies')
```

Reply:
Done.